### PR TITLE
Enable compiler-rt

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

**Comment by serge-sans-paille** [comment]
@sylvestre / @mgorny : even for dynamically linked libraries, I need to add `-lgnu_s` to get the unwinding symbols. An alternative would be to specify an unwinding library to use during testing at configure time. What do you think?

**Comment by serge-sans-paille** [comment]
Update: using `-lgcc_eh` is a lihter approach.

File: tests/whole-toolchain.cpp [hunk 1]
```diff
@@ -1,6 +1,9 @@
 // Tests all components of the toolchain
 // REQUIRES: clang, lld, compiler-rt, libc++
-// RUN: %clangxx -fuse-ld=lld -rtlib=compiler-rt -stdlib=libc++ %s -o %t
+//
+// NOTE: Adding -lgcc_eh here is required to provide unwinding informations. An
```
Comment (sylvestre) [inline tests/whole-toolchain.cpp:4]:
```suggestion
// NOTE: Adding -lgcc_eh here is required to provide unwinding information. An
```

File: tests/whole-toolchain.cpp [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+// alternative would be to force usage of LLVM unwinder when building compiler-rt.
```
Comment (sylvestre) [inline tests/whole-toolchain.cpp:5]: Yeah, as a next step, we should

**Review by sylvestre — APPROVED** [review]

**Comment by serge-sans-paille** [comment]
@mgorny: this is probably the closest we can have for libc++ combined with compiler-rt

**Comment by sylvestre** [comment]
@serge-sans-paille still WIP ?

**Pull request merged**